pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -303,3 +303,15 @@ def test_groupby_agg_coercing_bools():
     result = gp['c'].aggregate(lambda x: x.isnull().all())
     expected = Series([True, False], index=index, name='c')
     tm.assert_series_equal(result, expected)
+
+
+def test_order_aggregate_multiple_funcs():
+    # GH 25692
+    df = pd.DataFrame({'A': [1, 1, 2, 2], 'B': [1, 2, 3, 4]})
+
+    res = df.groupby('A').agg(['sum', 'max', 'mean', 'ohlc', 'min'])
+    result = res.columns.levels[1]
+
+    expected = pd.Index(['sum', 'max', 'mean', 'ohlc', 'min'])
```

Comment:
For my curiosity, what was the old order for various python versions?

Reply:
Index(['max', 'mean', 'min', 'ohlc', 'sum'], dtype='object'). This would have been the result in all python versions previous to the changes I made in concat (#25224). After the changes, this would have been the return for the versions below Python 3.6, and the order observed in this PR for 3.6 and above. This is why we converted dicts to OrderedDicts in the aggregation functions, to ensure that result would be consistent in all versions.